website/docs/guides/migration/from-v2-to-v3.md

```diff
@@ -0,0 +1,204 @@
+---
+sidebar_position: 1
+---
+
+# From v2 to v3
+
+KafkaFlow version 3 brings several significant changes and improvements. This guide will help you navigate through the migration process from version 2 to version 3.
+
+## Breaking Changes
```

Comment:
The first step needs to be the package references upgrade.
Example: https://learn.microsoft.com/en-us/aspnet/core/migration/70-80?view=aspnetcore-8.0&tabs=visual-studio-code#update-package-references

Reply:
It adds some verbosity to the document because of the number of packages, but I added it anyway so we can check the final result and evaluate it. Please check.